app/js/states/inventory/inventory.controller.js

```diff
@@ -298,40 +300,40 @@ function InventoryCtrl(
     function order() {
         $scope.systems = $filter('orderBy')($scope.systems,
             ['_type',
-            ($scope.reverse ?
-                '-' + $scope.predicate :
-                $scope.predicate)]);
+            ($scope.sorter.reverse ?
+                '-' + $scope.sorter.predicate :
+                $scope.sorter.predicate)]);
```

Comment:
$filter is a local sort. It depends on us having the complete systems data set. Inventory uses the api for paging so we only have one page at a time. We should call getData here.